modules/awstats.py

```diff
@@ -124,9 +126,9 @@ def configure(domain, logfile, period="hourly"):
                                        awstats,
                                        awstats_static)
 
-        with open(awstats_scr_path, 'w') as f:
+        with salt.utils.fopen(awstats_scr_path, 'w') as f:
             f.write(update)
-            os.chmod(awstats_scr_path, 0700)
+            os.chmod(awstats_scr_path, 0o700)
```

Comment:
You got an `o` in the middle there, not sure if that was intended

Reply:
This was suggested by the `Incompatible Python 3 code found. Proposed fix:`... warning from pylint w/ Salt's rules.
With Python3, you need to qualify the mode as being octal. The syntax works in 2.x as well.

https://docs.python.org/3/library/pathlib.html#basic-use
https://stackoverflow.com/questions/15607903/python-module-os-chmodfile-664-does-not-change-the-permission-to-rw-rw-r-bu

Reply:
Ah, OK, never seen that before, I just thought it may have been an entry when pressing `o` in vi or something ;)